skyportal/models.py

```diff
@@ -392,6 +391,8 @@ class Photometry(Base):
 
     origin = sa.Column(sa.String, nullable=True)
 
+    bulk_upload_id = sa.Column(sa.String, nullable=True)
```

Comment:
Could this be assigned by `PhotometryHandler.post`?

Reply:
it is created in `PhotometryHandler.post`

Reply:
😅 misunderstanding related to the issue of `/upload_photometry` being different from the api endpoint